apis/kueue/v1beta1/delegateconfig_types.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DelegateConfigSpec defines the desired state of DelegateConfig
+type DelegateConfigSpec struct {
```

Comment:
@mwielgus @trasc I would like to see some alignment of the API between the KEP and the WIP implementation, currently the types are disjoint. Possibly they correspond to the same concepts, but it's not obvious.

KEP types for reference: https://github.com/kubernetes-sigs/kueue/blob/20263b425c896a16cf4623a93cfd40633ea3213d/keps/693-multikueue/README.md?plain=1#L114-L154

Reply:
The rename is done, the two should (this and KEP should look more alike)